bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -901,6 +916,10 @@
 		<label>Filter Duration</label>
 		<description>Uptime of the filter</description>
 		<category>Time</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
As this rather *counts* the hours of operation since last reset of the value, maybe `Status` suits better here.

Reply:
Ok.

Reply:
Applied to three channel types